pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
+
+  return (
+    <div className={styles.canvas}>
+      <NavBar currentPage='Home' />
+      <div className={styles.page}>
```

Comment:
can you make this a `main` instead of a `div` its the main page content, it also follows the `homepage`.